src/SamplesApp/SamplesApp.UITests/TestFramework/ActivePlatformsAttribute.cs

```diff
@@ -26,9 +27,41 @@ public Platform[] Platforms
 			}
 		}
 
+		public Exception Error { get; }
+
+		public ActivePlatformsAttribute(Platform platform, [CallerArgumentExpression("platform")] string expression = "")
+		{
+		    if (expression.Length == 0)
+			{
+				Error = new NotSupportedException("'CallerArgumentExpression' is not supported?");
+			}
+
+			if (expression.Contains('|'))
+			{
+				// See https://github.com/unoplatform/uno/issues/9798 for info.
+				Error = new ArgumentException("Don't use bitwise or in ActivePlatforms attribute.");
+			}
+
+			Properties.Add("ActivePlatforms", new[] { platform });
+		}
+
 		public ActivePlatformsAttribute(params Platform[] platforms)
 		{
 			Properties.Add("ActivePlatforms", platforms);
 		}
 	}
 }
+
+namespace System.Runtime.CompilerServices
+{
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = true, Inherited = false)]
+    public sealed class CallerArgumentExpressionAttribute : Attribute
```

Comment:
Can you move this to separate file?

Reply:
Sure. Will do

Reply:
@Youssef1313, @MartinZikmund I think the correct changes were done yesterday right? 
This PR is missing something else or it can be merged?

Reply:
@agneszitte-nventive I think it should be ready to merge